### ISSUE TITLE
NVDA is very repetitive when entering dialogs in IE

### ISSUE THREAD

**Issue by aminimalanimal** [nvaccess/nvda#6659]
In the following codepen, I have an example of a dialog with aria-labelledby pointing to a heading within. When the user clicks the button to enter the dialog, the heading is read three times in IE environments. The heading is read (appropriately) twice in Firefox and Chrome—once because of the new landmark context and once because the user is focused on it.
Uncertain why NVDA in IE is extra repetitive in this case.

[Demonstration](http://codepen.io/aminimalanimal/pen/bBZYjY)

**Comment by feerrenrut** [comment]
Since the severity of this issue is low (though annoying) I'll set this to a priority 3 to investigate the cause of this.

**Comment by Adriani90** [comment]
@aminimalanimal unfortunately your link does not work anymore. Could you please test with NVDA 2019.1 RC1 if the issue is still occuring? Thanks!

**Comment by aminimalanimal** [comment]
I've updated the link. I don't currently have access to IE 11.

**Comment by Brian1Gaff** [comment]
That page it says does not work in the browser. In that case IE11, it 
actually says code pen does not work, so I guess they changed the code to 
stop IE folk using it anyway.


bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by Adriani90** [comment]
I cannot see any difference now with NVDA 2020 Beta 2 and IE11. The dialogs are very repetitive in every browser, but there is nothing that seems specific for IE. I am closing in favor of #8971. Please feel free to comment on this if I missed something and we can reopen.
